app/js/controllers/relatedCtrl.js

```diff
@@ -21,6 +22,7 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		$scope.entryListChange = function(){
 			// unselect playlist ID
 			$scope.playerData.config.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].useContext = false;
 			$scope.playlistId = {id:'',text:''};
 			// set entries list
 			$scope.playerData.config.plugins["related"].entryList = $scope.entryList;
```

Comment:
entry list gets an array of objects
see https://github.com/kaltura/playkit-js-related/blob/master/src/types/config.ts#L7